rudder-jetty/SOURCES/jetty-init-prevent-false-failed-starts.patch

```diff
@@ -0,0 +1,65 @@
+--- jetty7/bin/jetty.sh	2014-11-21 18:07:18.054680154 +0100
++++ jetty7/bin/jetty.sh	2014-11-21 18:09:53.251459474 +0100
+@@ -117,10 +117,10 @@
+   do
+     sleep 4
+     [ -z "$(grep STARTED $1 2>/dev/null)" ] || return 0
++
+     [ -z "$(grep STOPPED $1 2>/dev/null)" ] || return 1
+     [ -z "$(grep FAILED $1 2>/dev/null)" ] || return 1
+-    local PID=$(cat "$2" 2>/dev/null) || return 1
+-    kill -0 "$PID" 2>/dev/null || return 1
```

Comment:
Because I do throw away the PID handling logic, I do not use it anymore (ps is enough)